src/pages/workspaces/workspace/SubmissionHistory.js

```diff
@@ -214,6 +237,11 @@ const SubmissionHistory = _.flow(
     };
   });
 
+  useEffect(() => {
+    refresh();
```

Comment:
How hard would it be to update - and read from - the browser url when date ranges change? This would allow end users to link directly to, or bookmark, specific date ranges.

This is definitely a stretch goal; I wouldn't hold this PR too long to get that working.